app/controllers/reviews_controller.rb

```diff
@@ -161,6 +161,11 @@ def vote_down
       current_user.vote_against(@review)
     end
 
+    if ((@review.votes_against > -9) && @review.comment != "")
```

Comment:
Are the votes_against displayed as negative numbers? Just not sure.

It might also be good to check instead if its `>= -10` just so it's completely clear we're deleting it at 10 downvotes.